package.json

```diff
@@ -100,6 +100,7 @@
     "cookieconsent": "3.1.1",
     "cookies": "0.8.0",
     "css-to-react-native": "3.0.0",
+    "csstype": "^3.0.3",
```

Comment:
Please use fixed versions.

I recommend running `npm config set save-exact=true`, see https://stackoverflow.com/questions/40105268/how-to-force-npm-to-install-specific-version-of-package-by-default